scripts/run_frontend_tests.py

```diff
@@ -79,6 +80,7 @@ def run_dtslint_type_tests():
            TYPESCRIPT_DIR_RELATIVE_PATH]
     task = subprocess.Popen(cmd, stdout=subprocess.PIPE)
     output_lines = []
+    assert task.stdout is not None
```

Comment:
Please add an explanatory comment. Ditto below.

Reply:
I added a few comments. PTAL and resolve it if it looks good to you. Thanks,

Reply:
I think that if you add `if` we don't need the comment as the error message can explain it instead.

Reply:
As we are passing `stdout=subprocess.PIPE` as an argument, here `task.stdout` cannot be `None`. Also, as explained [here](https://docs.python.org/3.8/library/subprocess.html#subprocess.CompletedProcess), it may return `None` value in other cases. So, I think, we should keep this `assert`. please correct me if I am wrong.

Reply:
Okay, seems correct.